preview_generator/preview/builder/image__wand.py

```diff
@@ -95,8 +95,8 @@ def build_jpeg_preview(
         preview_name: str,
         cache_path: str,
         page_id: int,
-        extension: str = ".jpeg",
-        size: ImgDims = None,
```

Comment:
Is there here any reason to remove default values ?

Reply:
See previous comments.